schemas/rdf/rdf-ontology.ttl

```diff
@@ -1,2058 +1,2058 @@
-.
-
-###  https://admin-shell.io/aas/3/0/SubmodelElementList/value
-<https://admin-shell.io/aas/3/0/SubmodelElementList/value> rdf:type owl:ObjectProperty ;
-    rdfs:label "has value"^^xs:string ;
-    rdfs:domain aas:SubmodelElementList ;
-    rdfs:range aas:SubmodelElement ;
-    rdfs:comment "Submodel element contained in the list."@en ;
-.
-
-###  https://admin-shell.io/aas/3/0/SubmodelElementList/valueTypeListElement
-<https://admin-shell.io/aas/3/0/SubmodelElementList/valueTypeListElement> rdf:type owl:ObjectProperty ;
-    rdfs:label "has value type list element"^^xs:string ;
-    rdfs:domain aas:SubmodelElementList ;
-    rdfs:range aas:DataTypeDefXSD ;
-    rdfs:comment "The value type of the submodel element contained in the list."@en ;
-.
-
-###  https://admin-shell.io/aas/3/0/ValueList
-aas:ValueList rdf:type owl:Class ;
-    rdfs:label "Value List"^^xs:string ;
-    rdfs:comment "A set of value reference pairs."@en ;
-.
-
-###  https://admin-shell.io/aas/3/0/ValueList/valueReferencePairs
-<https://admin-shell.io/aas/3/0/ValueList/valueReferencePairs> rdf:type owl:ObjectProperty ;
-    rdfs:label "has value reference pairs"^^xs:string ;
-    rdfs:domain aas:ValueList ;
-    rdfs:range aas:ValueReferencePair ;
-    rdfs:comment "A pair of a value together with its global unique id."@en ;
-.
-
-###  https://admin-shell.io/aas/3/0/ValueReferencePair
-aas:ValueReferencePair rdf:type owl:Class ;
-    rdfs:label "Value Reference Pair"^^xs:string ;
-    rdfs:comment "A value reference pair within a value list. Each value has a global unique id defining its semantic."@en ;
-.
-
-###  https://admin-shell.io/aas/3/0/ValueReferencePair/value
-<https://admin-shell.io/aas/3/0/ValueReferencePair/value> rdf:type owl:DatatypeProperty ;
-    rdfs:label "has value"^^xs:string ;
-    rdfs:domain aas:ValueReferencePair ;
-    rdfs:range xs:string ;
-    rdfs:comment "The value of the referenced concept definition of the value in 'valueID'."@en ;
-.
-
-###  https://admin-shell.io/aas/3/0/ValueReferencePair/valueID
-<https://admin-shell.io/aas/3/0/ValueReferencePair/valueID> rdf:type owl:ObjectProperty ;
-    rdfs:label "has value ID"^^xs:string ;
-    rdfs:domain aas:ValueReferencePair ;
-    rdfs:range aas:Reference ;
-    rdfs:comment "Global unique id of the value."@en ;
+@prefix aas: <https://admin-shell.io/aas/3/0/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xs: <http://www.w3.org/2001/XMLSchema#> .
+@base <https://admin-shell.io/aas/3/0/> .
+
+<https://admin-shell.io/aas/3/0/> rdf:type owl:Ontology ;
+    owl:versionInfo "V3.0-work-in-progress" ;
+    rdfs:comment "This ontology represents the data model for the Asset Administration Shell according to the specification 'Details of the Asset Administration Shell - Part 1 - Version V3.0-work-in-progress'."@en ;

```

Comment:
Can we change bot the `versionInfo` and the comment to "V3.0" and "Specification of the Asset Administration Shell: Part 1"?

Reply:
I fixed it in https://github.com/aas-core-works/aas-core-meta/pull/268.

I changed the comment to a more general one, so that it survives future versions of the specs: https://github.com/aas-core-works/aas-core-codegen/pull/381.

In particular, see:
* https://github.com/aas-core-works/aas-core-codegen/pull/381/files#diff-4a6f81ac34dc84c7116ed5bc6285a39d2f4d8a12344526951b77b681f285a672 , and 
* https://github.com/aas-core-works/aas-core-codegen/pull/381/files#diff-8c8150f399182260bfcf8beca1530bf28db4150bbaf6dae9169f0d9c8eeaacb3.